basics/docs-style.md

```diff
@@ -0,0 +1,201 @@
+---
+layout: contribute
+title: Docs style
+---
+
+# Bazel docs style guide
+
+Thank you for contributing to Bazel's documentation. This serves as a quick
+documentation style guide to get you started. For any style questions not
+answered by this guide, follow the
+[Google developer documentation style guide](https://developers.google.com/style).
+
+## Defining principles
+
+Bazel docs should uphold these principles:
+
+-  **Concise.** Use as few words as possible.
+-  **Clear.** Use plain language. Write without jargon for a fifth-grade
+   reading level.
+-  **Consistent.** Use the same words or phrases for repeated concepts
+   throughout the docs.
+-  **Correct.** Write in a way where the content stays correct for as long as
+   possible by avoiding time-based information and promises for the future.
+
+## Writing
+
+This section contains basic writing tips.
+
+### Headings
+
+-  Page-level headings start at H2. (H1 headings are used as page titles.)
+-  Make headers as short as is sensible. This way, they fit in the TOC
+   without wrapping.
+   -  **YES:** Permissions
+   -  **NO:** A brief note on permissions
+
+-  Use sentence case for headings
+   -  **YES:** Set up your workspace
+   -  **NO:** Set Up Your Workspace
+
+-  Try to make headings task-based or actionable. If headings are conceptual,
+   it may be based around understanding, but write to what the user does.
+   -  **YES:** Preserving graph order
+   -  **NO:** On the preservation of graph order
+
+### Names
+
+-  Capitalize proper nouns, such as Bazel and Starlark.
+   -  **YES:** At the end of the build, Bazel prints the requested targets.
+   -  **NO:** At the end of the build, bazel prints the requested targets.
+
+-  Keep it consistent. Don't introduce new names for existing concepts. Where
+   applicable, use the term defined in the
+   [Glossary](https://docs.bazel.build/versions/3.4.0/glossary.html#command).
```

Comment:
Should we use https://docs.bazel.build/versions/3.4.0/glossary.html or https://docs.bazel.build/versions/master/glossary.html or https://docs.bazel.build/glossary.html ?

Reply:
Thank you!